project-api/project-api-test/src/test/resources/projects_for_resolution_tests/package_with__http_plugin_compile_issue_in_tests/main.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/io;
```

Comment:
Lets remove this import

Reply:
Removed this exact case and added a test by adding a custom code analyzer plugin.